examples/searchlight/GenreSearchlight.py

```diff
@@ -0,0 +1,75 @@
+import numpy as np
```

Comment:
The file should be named like any other Python module, e.g., "genre_searchlight_example.py":
https://www.python.org/dev/peps/pep-0008/#naming-conventions